apps/frontend/core-dapp/src/components/TorusTrap.tsx

```diff
@@ -0,0 +1,93 @@
+import { FC, Suspense, useEffect, useState } from 'react'
+import { useIntercom } from './Intercom'
+
+export function useDetectTorusIframe(): HTMLIFrameElement | undefined {
+  const [torusIframe, setTorusIframe] = useState<HTMLIFrameElement | undefined>(undefined)
+
+  useEffect(() => {
+    const node = document.querySelector('iframe#torusIframe')
+    if (node instanceof HTMLIFrameElement) {
+      setTorusIframe(node)
+    }
+  }, [setTorusIframe])
```

Comment:
saw your latest commit to remove this but somehow this is still here?

```suggestion
  }, [])
```

Reply:
There were two of them, fixed both now